flowermd/library/systems.py

```diff
@@ -1,3 +1,37 @@
 """Examples for the Systems class."""
 
 # This is a placeholder for any class that inherits from base.system
+import numpy
```

Comment:
We should use `import numpy as np` since we use `np` in the code, and for consistency.